bowtie/_cli.py

```diff
@@ -153,29 +161,82 @@ def summary(input: Iterable[str], format: str | None):
         reverse=True,
     )
 
+    validity = []
+    for _, schemas in summary._combined.items():
+        schema = []
+        instances = []
+        for result in schemas["results"]:
+            validation = {}
+            for implementation in summary.implementations:
+                valid = result[1].get(implementation["image"], "error")
+                if valid == "error":
```

Comment:
I think this probably only handles 1 error case, which is why adding another test or two like I mentioned below is likely going to be helpful.

A test case can error either at the "case level" or at the "test level". In other words, if I have a schema: `{"foo": "bar"}` and I validate 3 instances against it, an implementation can error:

* by blowing up entirely at the schema `{"foo": "bar"}` because it has some bug interpreting that schema
* by blowing up for 1 of the 3 instances but not others, because it has some bug with a particular edge case

I *think* this may only handle one of the two cases but it'll be clearer if you try to write a test for each -- write a test where you put "crash:1" in the description of a test *case* that you pass to `envsonschema`, and a second test case where you put "crash:1" in the description of an individual *test* you pass but not in the test case. And then we'll be able to see whether we're handling both cases.

Let me know if that doesn't make sense to you!